__tests__/app.test.js

```diff
@@ -49,4 +49,73 @@ describe('GET api', () => {
     })
 })
             
-            
\ No newline at end of file
+ describe('GET /api/articles', () => {
+    test('GET 200: should return an array of objects containing all articles', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body}) => {
+            const articles = body.articles;
+            expect(articles).toHaveLength(13);
+            articles.forEach((article) => {
+                expect(article).toMatchObject({
+                    article_id: expect.any(Number),
+                    title: expect.any(String),
+                    topic: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    comment_count: expect.any(Number)
+                })
+            })
+        })
+    })
+ })
+
+ describe('GET /api/articles/:article_id', () => {
+     test('GET 200: sends a single article to the client when given an appropriate article_id', () => {
+         return request(app)
+         .get('/api/articles/1')
+         .expect(200)
+         .then(({body}) => {
+            const article = body.article;
+            expect(article).toMatchObject({
+              article_id: expect.any(Number),
```

Comment:
I would check that the article id is 1 here, proving that you have the correct article.